temba/msgs/models.py

```diff
@@ -1506,6 +1539,18 @@ def create_outgoing(cls, org, user, recipient, text, broadcast=None, channel=Non
         if channel:
             analytics.gauge('temba.msg_outgoing_%s' % channel.channel_type.lower())
 
+        metadata = None
+        if quick_replies:
+            for counter, reply in enumerate(quick_replies):
+                (value, errors) = Msg.evaluate_template(text=reply, context=expressions_context, org=org)
+                if value:
+                    if contact.is_test:
+                        quick_replies[counter] = value
+                    else:
+                        channel_type = Channel.get_type_from_code(channel.channel_type)
```

Comment:
As we said previously - this truncation should be happening in the channel handler